### ISSUE TITLE
Bringing old code into current standards

### ISSUE THREAD

**Issue by dextermb** [langleyfoxall/technologies#135]
### What you would like to change/add
I'd like to add a clause that states that when working around old areas of a codebase that the old code should be updated to meet currently internal standards.

### Why you would like to change/add this
This will generally help make code maintainable and easier to read if extra development is required in the future.

While making development take longer I believe the benefit outweighs the negatives.

### Examples
Say you're working on tweaking an old function, if the code within the same scope is no longer in line with the current standard then it should be updated while you're working on the new functionality.

Here's a snippet from a recent project, if I wanted to add something to the first condition in the if-statement then I should also update the function to match current JS standards.

```js
handleBackPress() {
    if (this.state.editingNotes || this.state.editingCannotComplete || this.state.siteMapVisible || this.state.damageFlagVisible) {
        this.cancelEditingNotes();
        this.cancelEditingCannotComplete();
        this.cancelSiteMap();
        this.cancelDamageFlag();
    } else {
        if (!this.state.answersChanged) {
            BackHandler.removeEventListener("hardwareBackPress", this.handleBackPress);
            this.handleGoBackPress();
        } else {
            this.toggleBackModal();
        }
    }
    return true;
}
```

This that I would change here:
* Destructing
* Early exiting
* Single quotes for strings

As we also have PHP standards this can also be done with older PHP code.

### Checklist
- [x] I have checked that this is not a duplicate issue.

**Comment by AlexCatch** [comment]
I partly agree - in an ideal world that's exaclty what needs to happen but in the commerical world it's often just not possible, expecially when dealing with expansive awful code bases.

As long as we define the scope of changes to just functions then it sounds good.

**Comment by dextermb** [comment]
> I partly agree - in an ideal world that's exaclty what needs to happen but in the commerical world it's often just not possible, expecially when dealing with expansive awful code bases.
> 
> As long as we define the scope of changes to just functions then it sounds good.

I agree, it'd be brilliant if we could spend time updating whole codebases but I feel like that would not be commercially possible hence why I mention it would be within the code of the code you're working on.
